Sources/GRPCProtobuf/InputParser.swift

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
This shouldn't be in this module, this module is for protobuf components which are used at runtime (i.e. when running a client or server). This is used at code generation time so should be in its own module.